autoscheduler/frontend/src/components/GenericSnackbar.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Snackbar, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+interface GenericSnackbarProps {
+  snackbarMessage: string;
+  setSnackbarMessage: (message: string) => void;
+}
+
+const GenericSnackbar: React.FC<GenericSnackbarProps> = ({
```

Comment:
Why did you choose to have multiple snackbars rather than use a single one and update the message using redux? I doubt anyone will actually encounter this so it's not a huge deal (and idk if it's worth bothering to fix unless we end up adding more snackbars in the future), but if you cause an error with generating schedules and click a meeting card with no corresponding course card, it will make both snackbars appear at once:
![image](https://user-images.githubusercontent.com/28655462/121785614-58bd8580-cb80-11eb-90a0-1e0a7655f194.png)

Reply:
I thought we had discussed about it but maybe we didn't - but it was just much simpler to make them both use a Generic snackbar than doing it through Redux.

I honestly hadn't even considered this had a possibility of happening so good catch, we can definitely switch it to doing redux if we think it's necessary

Reply:
Nah, like I said I don't think anyone will ever actually do that so it's not worth the effort